<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComptagesSettings</class>
 <widget class="QDialog" name="ComptagesSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="db_host"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Nom DB</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="db_name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Porte</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="db_port"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nom d'utilisateur</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="db_username"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Mot de passe</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="db_password">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="extra_layers">
     <property name="text">
      <string>montrer les couches suppl√©mentaires</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ComptagesSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ComptagesSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
